src/screens/AboutPage.tsx

```diff
@@ -0,0 +1,64 @@
+import { StatusBar } from 'expo-status-bar';
```

Comment:
This file should probably be named just About.tsx

Reply:
I've updated this but wont do another push until tomorrow after getting other input.

Reply:
I updated it again to Info instead of about. Thank goodness there's a search feature lol.